### PR TITLE
Enhancement: Rename and reorder ISL list table columns

### PR DESCRIPTION
**Pull request by neeraj890kumar**
- Fix Flow path and validate flow should not be visible if flow is not active on controller
- Fix loader text changes
- Fix flow contract screen search design changes

Resolves: #1739 , #1672

### CODE REVIEW

File: services/src/openkilda-gui/ui/src/app/modules/flows/flow-detail/flow-detail.component.html [hunk 1]
```diff
@@ -224,23 +224,30 @@ <h6 class="card-title card-custom-title">FLOW DETAILS</h6>
...
-                   
-                   <p class="col-md-6 alert alert-warning" *ngIf="(descrepancyData.status.inventory == '-' ) ">
-                    <strong>*Note:</strong> Flow is not active in inventory</p>
-  
-            </div>
           </div>
-          
+          <div class="note-container">
+            <p class="col-md-6 alert alert-warning" *ngIf="(descrepancyData.status.controller =='-' && descrepancyData.status.inventory == 'Active')">
+               <strong>*Note:</strong> Flow is not active in kilda controller</p>
```
Comment (gauravchug) [inline services/src/openkilda-gui/ui/src/app/modules/flows/flow-detail/flow-detail.component.html]: Flow doesn't exist in the controller

File: services/src/openkilda-gui/ui/src/app/modules/flows/flow-detail/flow-detail.component.html [hunk 2]
```diff
@@ -224,23 +224,30 @@ <h6 class="card-title card-custom-title">FLOW DETAILS</h6>
...
+               
+               <p class="col-md-6 alert alert-warning" *ngIf="(descrepancyData.status.inventory == '-' ) ">
+                <strong>*Note:</strong> Flow is not active in inventory</p>
```
Comment (gauravchug) [inline services/src/openkilda-gui/ui/src/app/modules/flows/flow-detail/flow-detail.component.html]: Same for inventory, if the *flow doesn't exist in the inventory* then show this message or if flow exists in inventory but not active in inventory then you can show this message.
*Flow is not active in inventory*

Comment (neeraj890kumar) [reply services/src/openkilda-gui/ui/src/app/modules/flows/flow-detail/flow-detail.component.html]: resolved

File: services/src/openkilda-gui/ui/src/app/modules/flows/flow-detail/flow-detail.component.html (hunk 1 above)
Comment (neeraj890kumar) [reply services/src/openkilda-gui/ui/src/app/modules/flows/flow-detail/flow-detail.component.html]: resolved

**Pull request merged**